mkt/commonplace/templates/commonplace/index_react.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
Yeah, sorry, I know that's annoying too...but a few small changes like that, and suddenly our process for deploying React is completely opaque.